README.md

```diff
@@ -21,7 +21,7 @@ high performance API-first projects. Extend or override everything you want.
 ## Documentation
 
 The documentation of API Platform Core is available online and can be edited in
-[its dedicated repository](https://github.com/api-platform/doc/tree/master/core).
+[its dedicated repository](https://github.com/api-platform/doc/tree/master/index.md).
```

Comment:
Should now be:

`The documentation of API Platform can be browsed [on the official website](https://api-platform.com/docs/).`